otx/api/entities/inference_parameters.py

```diff
@@ -36,7 +36,6 @@ class InferenceParameters:
     is_evaluation: bool = False
     update_progress: Callable[[int, Optional[float]], Any] = default_progress_callback
 
-    # TODO(negvet): use separate ExplainParameters dataclass for this
     explainer: str = ""
```

Comment:
If we use `ExplainParameters`, how about removing `explainer`?

Reply:
Thanks for noticing!
`explainer` is a placeholder for a default explain algorithm, we will use it in the next iteration of revision. Don't you mind to keep it here for a while?

Reply:
No problem to keep